FirebaseSegmentation/Sources/Public/FIRSegmentation.h

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2019 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@class FIRApp;
+/**
+ *  The Firebase Segmentation SDK is used to associate a custom, non-Firebase custom installation
+ * identifier to Firebase. Once this custom installation identifier is set, developers can use the
+ * current app installation for segmentation purposes. If the custom installation identifier is
+ * explicitely set to nil, any existing custom installation identifier data will be removed.
+ */
+NS_SWIFT_NAME(Segmentation)
+@interface FIRSegmentation : NSObject
+
+/// Firebase Remote Config service fetch error.
```

Comment:
nit: this should say Firebase Segmentation instead of Firebase Remote Config

Reply:
good find. Thank you!